google/cloud/security/common/gcp_type/gce_network.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A Enforced Network Resource."""
+
+
+# pylint: disable=too-few-public-methods
+class GceNetwork(object):
```

Comment:
I think this would be better to be named as ```InstanceNetworkInterface```.  Can you please update this everythere?